src/custom/state/price/updater.ts

```diff
@@ -140,7 +133,10 @@ export default function FeesUpdater(): null {
 
   // Update if any parameter is changing
   useEffect(() => {
-    // Don't refetch if window is not visible, or some parameter is missing
+    // Don't refetch if:
+    //  - window is not visible
+    //  - browser is offline
```

Comment:
Where is the check for the offline flag?

Reply:
Later! it was originally there but was moved (i left the comment)